src/style.css

```diff
@@ -1 +1,52 @@
-/* Styles go here */
+
+body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  --hover-active-color: #00acdc;
+}
+
+.header {
+  box-sizing: border-box;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  max-width: 1200px;
+  min-width: 1024px;
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 3px 50px 1px;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__link {
+  text-decoration: none;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+}
+
+.nav__link:hover {
+  color: var(--hover-active-color);
+}
+
+.is-active {
+  color: var(--hover-active-color);
+}
+
+.is-active::after {
+  content: "";
+  background-color: var(--hover-active-color);
+  height: 4px;
+  border-radius: 8px;
+  display: block;
+  position: relative;
+  bottom: -17px;
```

Comment:
it is bad to hardcode the position. When you will fix the height of the links, you can use `position: absolute` here with `bottom: 0` and it will work properly on any height change